app/scripts/services/rest.js

```diff
@@ -91,7 +91,10 @@ angular.module('openhimConsoleApp')
       }),
 
       // Visualizer API
-      Visualizers: $resource(server + '/visualizers/:name', { name: '@name' }),
+      Visualizers: $resource(server + '/visualizers/:name', { name: '@name' }, {
+        save: { method: 'POST' },
+        query: { method: 'GET', isArray: true }
```

Comment:
I think these are the defaults of save and query, so this object can be removed.